### DISCUSSION TITLE
How do I resize the formulas displayed in MathPreviewPanel

### DISCUSSION THREAD

**Discussion by saxyx** [James-Yu/LaTeX-Workshop#4362] [Q&A] [answered]
The size of the current display is difficult to see
![image](https://github.com/user-attachments/assets/7ed6e083-6c61-49e3-864d-9d65d5082eba)

**Comment by riasat-sheikh** [comment, marked as answer]
You need to change the value for this `latex-workshop.hover.preview.scale`. The default value is `1`. You can change it to `1.5` maybe. For more information refer to the [wiki](https://github.com/James-Yu/LaTeX-Workshop/wiki/Hover#latex-workshophoverpreviewscale).

**Reply by saxyx** [reply]
Thank you very much! This parameter was unintentionally set to 0.2, and after changing it to 1, it returned to its normal size.
